frame/alliance/src/tests.rs

```diff
@@ -441,9 +441,43 @@ fn retire_works() {
 			member: (3),
 			unreserved: None,
 		}));
+
+		// Move time on:
+		System::set_block_number(System::block_number() + RetirementPeriod::get());
+
+		assert_powerless(Origin::signed(3));
 	});
 }
 
+fn assert_powerless(user: Origin) {
```

Comment:
Nice!

I think right not it make sense to check if method supposed to be executed by a member.
For `add_unscrupulous_items` for example, its not really a case, because its checked by `AnnouncementOrigin` provided by client of pallet.

If you think it should be different (should check if the caller is a member for example), I think we need a different issue for it.